workers/src/main/scala/workers/ItcServer.scala

```diff
@@ -25,10 +25,12 @@ import japgolly.scalajs.react.callback.*
 import log4cats.loglevel.LogLevelLogger
 import lucuma.ags.Ags
 import lucuma.ags.AgsAnalysis
+import lucuma.itc.client.ItcClient
```

Comment:
I think you don't need the `clue` imports anymore (lines 11-14). However, I suggest changing the `lucuma-itc` API so that it receives the `clue` client directly instead of building it from a `http4s` client. This will make it the API independent from `http4s` and clients are free to use it or not.

Reply:
I guess it would mean changing it here right?
https://github.com/gemini-hlsw/lucuma-itc/blob/main/modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala#L50

`lucuma-itc` is in use by the odb as well. Would it be fine to do this on the jvm side as well?

Reply:
Yes, it should be OK.

Reply:
I'll do that in a separate PR as it needs an update on the odb side